ome_zarr/reader.py

```diff
@@ -262,10 +262,17 @@ def matches(zarr: ZarrLocation) -> bool:
     def __init__(self, node: Node) -> None:
         super().__init__(node)
 
+        axes_values = {"t", "c", "z", "y", "x"}
         try:
             multiscales = self.lookup("multiscales", [])
-            version = multiscales[0].get("version", "0.1")
+            version = multiscales[0].get(
+                "version", "0.1"
+            )  # should this be matched with Format.version?
             datasets = multiscales[0]["datasets"]
+            # axes field was introduced in 0.3, before all data was 5d
+            axes = tuple(multiscales[0].get("axes", ["t", "c", "z", "y", "x"]))
+            if len(set(axes) - axes_values) > 0:
+                raise RuntimeError(f"Invalid axes names: {set(axes) - axes_values}")
```

Comment:
As suggested in https://github.com/ome/ome-zarr-py/pull/89#issuecomment-879789276, passing axes data to `napari-ome-zarr`

```suggestion
                raise RuntimeError(f"Invalid axes names: {set(axes) - axes_values}")
            node.metadata['axes'] = axes
```

Reply:
Isn't that metadata already in the node, @will-moore? Is this just a case of needing/wanting an accessor?

Reply:
Can I access it from there as in https://github.com/ome/napari-ome-zarr/pull/9/files ?

Reply:
I would have _thought_ so. What all is in `metadata.keys()` at the moment?

Reply:
In `napari-ome-zarr`, the `node.metadata` (without the suggested change above) is:
```
metadata dict_keys(['name', 'visible', 'contrast_limits', 'colormap'])
```

Reply:
When I add this line I get the following issue in napari:
```
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/components/viewer_model.py", line 1003, in _add_layer_from_data
    layer = add_method(data, **(meta or {}))
TypeError: add_image() got an unexpected keyword argument 'axes'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pape/Work/software/conda/miniconda3/envs/main/bin/napari", line 10, in <module>
    sys.exit(main())
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/__main__.py", line 420, in main
    _run()
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/__main__.py", line 309, in _run
    **kwargs,
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/view_layers.py", line 169, in view_path
    return _make_viewer_then('open', args, kwargs)
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/view_layers.py", line 119, in _make_viewer_then
    method(*args, **kwargs)
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/components/viewer_model.py", line 857, in open
    _path, kwargs, plugin=plugin, layer_type=layer_type
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/components/viewer_model.py", line 929, in _add_layers_with_plugins
    added.extend(self._add_layer_from_data(*_data))
  File "/home/pape/Work/software/conda/miniconda3/envs/main/lib/python3.7/site-packages/napari/components/viewer_model.py", line 1015, in _add_layer_from_data
    ) from exc
TypeError: _add_layer_from_data received an unexpected keyword argument ('axes') for layer type image
```

Reply:
with or without https://github.com/ome/napari-ome-zarr/pull/9 ? (If without, then it would be interesting to know whether or not we can be more [liberal](https://devopedia.org/postel-s-law) in accepting old and new inputs.

Reply:
Ah, sorry never mind; I tried with the wrong branch.

Reply:
> with or without [ome/napari-ome-zarr#9](https://github.com/ome/napari-ome-zarr/pull/9)

Yes, I have tried with https://github.com/ome/napari-ome-zarr/pull/8 instead and there it fails with the error from above.
I haven't tried with nr 9 yet.

Reply:
I get the same issue with nr 9.

Reply:
Works now.